app/views/participants/fields/_financial.html.erb

```diff
@@ -5,7 +5,7 @@
 		<%= f.input :free_reduced_lunch_signed_up, :label => "Signed up" %>
 	<% end %>
 
-	<%= f.input :household_size, :collection => 1..20, :as => :select, :label => "Number of persons in household", :hint => "Include the student in this count." %>
+	<%= f.input :household_size, :collection => 1..20, :as => :select, :label => "Number of persons in household"%>
```

Comment:
For these items where we're removing the help text from the code, we should create a migration and also populate `seeds.rb` so that current customers don't lose them.